roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -77,6 +82,21 @@
     resource_name: rolebinding-reader
     user: "system:serviceaccount:{{ openshift_logging_elasticsearch_namespace }}:aggregated-logging-elasticsearch"
 
+# needed for prometheus.  can remove other bindings?
+- oc_adm_policy_user:
+    state: present
+    namespace: "{{ openshift_logging_elasticsearch_namespace }}"
+    resource_kind: cluster-role
+    resource_name: cluster-reader
+    user: "system:serviceaccount:{{openshift_logging_elasticsearch_namespace }}:aggregated-logging-elasticsearch"
+
+- oc_adm_policy_user:
```

Comment:
this goes away if we can pass the additional CA as a parameter rather than having to copy it should we hold on that since this is a security change?

cc @simo5